xml/System.Runtime.InteropServices/RuntimeInformation.xml

```diff
@@ -184,7 +184,18 @@ The property returns a string that indicates the name of the currently executing
       <Docs>
         <summary>Gets the platform architecture on which the current app is running.</summary>
         <value>The platform architecture on which the current app is running.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This returned value is intended to represent the actual architecture of the underlying operating system. It is designed
+to ignore architecture emulation infrastructure that may be involved to run the process.
```

Comment:
```suggestion
The returned value is intended to represent the actual architecture of the underlying operating system. It is a best effort to ignore the architecture emulation infrastructure that may be involved to run the process. The returned value takes into account emulation built into Windows and macOS operation systems. The returned value does not take into account emulation using Qemu that is typically used on Linux operation system. 
```
It is difficult to get the actual OS architecture with Qemu - see the discussion in https://github.com/dotnet/runtime/pull/60910.